### ISSUE TITLE
Accessibility: Buttons must have discernible text

### ISSUE THREAD

**Issue by jose-cortina** [kyma-project/console#1434]
**Description**

Screen reader users are not able to discern the purpose of button that has no accessible name.

**Expected result**

Fix at least one of these issues:

- Element does not have inner text that is visible to screen readers
- aria-label attribute does not exist or is empty
- aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
- Element's default semantics were not overridden with role="presentation"
- Element's default semantics were not overridden with role="none"
- Element has no title attribute or the title attribute is empty

Reference: https://dequeuniversity.com/rules/axe/3.3/button-name

**Actual result**

Some buttons do not comply with the above criteria, e.g.

`<button class="fd-button--light sap-icon--search"></button>`

**Steps to reproduce**

Tested on https://console.gke-release-1-8-0-rc2.kyma.pro/home/workspace.

**Comment by parostatkiem** [comment]
Honestly, is there any non-zero chance that a blind person will be a Kyma administrator/user? I can't imagine a company hiring a blind person for managing Kyma... I think Kyma's Console is a special "website" precisely designed for certain people, unlike a "public" website, such as Facebook or Instagram.
It's like making a wheelchair ramp on the top of a high, rocky mountain. 
Let's not waste our workforce for a thing that will never be used ;)
